pinot-connectors/pinot-spark-common/src/main/scala/org/apache/pinot/connector/spark/common/PinotClusterClient.scala

```diff
@@ -33,7 +33,7 @@ import scala.util.{Failure, Success, Try}
  */
 private[pinot] object PinotClusterClient extends Logging {
   private val TABLE_SCHEMA_TEMPLATE = "http://%s/tables/%s/schema"
-  private val TABLE_BROKER_INSTANCES_TEMPLATE = "http://%s/v2/brokers/tables/%s"
+  private val TABLE_BROKER_INSTANCES_TEMPLATE = "http://%s/v2/brokers/tables/%s?state=%s"
```

Comment:
can you instead use the livebrokers API? We have used it at scale in the past so there are no concerns with whether it works or not.